### PR TITLE
Pull Request for Contoso Financial Assistant

### PR DESCRIPTION
**Pull request by shikha1970**
# Description

Checked in code that showcases the functionalities of Assistant API for a BFSI use case

# Checklist

- [x] I have read the [contribution guidelines](https://github.com/Azure-Samples/azureai-samples/blob/main/CONTRIBUTING.md)
- [x] I have coordinated with the docs team (mldocs@microsoft.com) if this PR deletes files or changes any file names or file extensions.
- [x] This notebook or file is added to the [CODEOWNERS](https://github.com/Azure-Samples/azureai-samples/blob/main/.github/CODEOWNERS) file, pointing to the author or the author's team.

### CODE REVIEW

**Comment by shikha1970** [comment]
Please help me merge this pull request - shagrawal@microsoft.com

**Comment by shikha1970** [comment]
I have revalidated and repo looks good to me.

**Comment by diondrapeck** [comment]
@shikha1970 Please follow the instructions in CONTRIBUTING.md to resolve the failing pre-commit gate: https://github.com/Azure-Samples/azureai-samples/blob/main/CONTRIBUTING.md#3-set-up-pre-commit

File: .gitmodules
Comment (kdestin) [inline .gitmodules:1]: Please remove this file, this PR doesn't appear to depend on git submodules

File: scenarios/Assistants/contoso_financial_assistant/LICENSE
Comment (kdestin) [inline scenarios/Assistants/contoso_financial_assistant/LICENSE:1]: Does this sample intentionally have a separate license from the rest of the samples in this repository?

File: scenarios/Assistants/contoso_financial_assistant/.gitignore
Comment (kdestin) [inline scenarios/Assistants/contoso_financial_assistant/.gitignore:1]: This gitignore is a duplicate of the gitignore in the root of this repository

File: scenarios/Assistants/function_calling/assistants_function_calling_with_bing_search.ipynb
Comment (kdestin) [inline scenarios/Assistants/function_calling/assistants_function_calling_with_bing_search.ipynb:1]: Is this change intentional? It appears out of scope of the rest of the changes introduced in this PR.

File: scenarios/Assistants/contoso_financial_assistant/README.md
Comment (kdestin) [inline scenarios/Assistants/contoso_financial_assistant/README.md:1]: Please take a read through our contributing guideline for guidance on the format that we expect for sample READMEs: https://github.com/Azure-Samples/azureai-samples/blob/main/CONTRIBUTING.md#write-your-contribution

Notably:

* This README is missing the yaml frontmatter that makes it discoverable on learn.microsoft.com
* The README does not follow the README template referenced in the contributing guide

**Review by kdestin — CHANGES_REQUESTED** [review]

File: scenarios/Assistants/function_calling/assistants_function_calling_with_bing_search.ipynb
Comment (shikha1970) [reply scenarios/Assistants/function_calling/assistants_function_calling_with_bing_search.ipynb:1]: reverted to original state. This was by mistake.

File: scenarios/Assistants/contoso_financial_assistant/LICENSE
Comment (kdestin) [reply scenarios/Assistants/contoso_financial_assistant/LICENSE:1]: I think you accidentally deleted the LICENSE in the root of the repository instead of the one introduced in this PR.

File: scenarios/Assistants/contoso_financial_assistant/.gitignore
Comment (kdestin) [reply scenarios/Assistants/contoso_financial_assistant/.gitignore:1]: I think you accidentally deleted the .gitignore in the root of the repository instead of the one introduced in this PR.

Comment (shikha1970) [reply scenarios/Assistants/contoso_financial_assistant/.gitignore:1]: added back.

File: scenarios/Assistants/contoso_financial_assistant/README.md
Comment (shikha1970) [reply scenarios/Assistants/contoso_financial_assistant/README.md:1]: fixed.

File: scenarios/Assistants/contoso_financial_assistant/LICENSE
Comment (shikha1970) [reply scenarios/Assistants/contoso_financial_assistant/LICENSE:1]: this is fixed now. Please see the LICENSE is moved to the root location.

File: scenarios/Assistants/contoso_financial_assistant/.gitignore
Comment (kdestin) [reply scenarios/Assistants/contoso_financial_assistant/.gitignore:1]: Please revert the deletion of the gitignore in the root of the repository

File: scenarios/Assistants/contoso_financial_assistant/README.md
Comment (kdestin) [reply scenarios/Assistants/contoso_financial_assistant/README.md:1]: This README is still missing most of the sections in the README template: https://github.com/Azure-Samples/azureai-samples/blob/main/scenarios/README-template.md

File: scenarios/Assistants/contoso_financial_assistant/README.md [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+Contributor License Agreement (CLA) declaring that you have the right to, and actually do, grant us
+the rights to use your contribution. For details, visit https://cla.opensource.microsoft.com.
+
+When you submit a pull request, a CLA bot will automatically determine whether you need to provide
+a CLA and decorate the PR appropriately (e.g., status check, comment). Simply follow the instructions
+provided by the bot. You will only need to do this once across all repos using our CLA.
+
+This project has adopted the [Microsoft Open Source Code of Conduct](https://opensource.microsoft.com/codeofconduct/).
+For more information see the [Code of Conduct FAQ](https://opensource.microsoft.com/codeofconduct/faq/) or
+contact [opencode@microsoft.com](mailto:opencode@microsoft.com) with any additional questions or comments.
```
Comment (kdestin) [inline scenarios/Assistants/contoso_financial_assistant/README.md:74]: This section is unneeded, it's already present in the repository's CONTRIBUTING.md

File: scenarios/Assistants/contoso_financial_assistant/README.md [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+
+## Trademarks
+
+This project may contain trademarks or logos for projects, products, or services. Authorized use of Microsoft 
+trademarks or logos is subject to and must follow 
+[Microsoft's Trademark & Brand Guidelines](https://www.microsoft.com/en-us/legal/intellectualproperty/trademarks/usage/general).
+Use of Microsoft trademarks or logos in modified versions of this project must not cause confusion or imply Microsoft sponsorship.
+Any use of third-party trademarks or logos are subject to those third-party's policies.
```
Comment (kdestin) [inline scenarios/Assistants/contoso_financial_assistant/README.md:82]: I don't believe we need this disclaimer.

File: LICENSE [hunk 3]
```diff
@@ -18,4 +18,4 @@ FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
 AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
 LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
 OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
-SOFTWARE.
+SOFTWARE.
```
Comment (kdestin) [inline LICENSE:21]:
```suggestion
SOFTWARE.

```

File: scenarios/Assistants/contoso_financial_assistant/README.md (hunk 2 above)
Comment (shikha1970) [reply scenarios/Assistants/contoso_financial_assistant/README.md]: Fixed

File: scenarios/Assistants/contoso_financial_assistant/README.md (hunk 1 above)
Comment (shikha1970) [reply scenarios/Assistants/contoso_financial_assistant/README.md]: Fixed

File: scenarios/Assistants/contoso_financial_assistant/.gitignore
Comment (shikha1970) [reply scenarios/Assistants/contoso_financial_assistant/.gitignore:1]: My apologies. restored now

**Comment by shikha1970** [comment]
I changed a few files that i should not have changed. So creating a fresh fork and pull request.
